pgml-dashboard/static/js/utilities/toast.js

```diff
@@ -1,35 +1,35 @@
 function createToast(message) {
-    const toastElement = document.createElement('div');
-    toastElement.classList.add('toast', 'hide');
-    toastElement.setAttribute('role', 'alert');
-    toastElement.setAttribute('aria-live', 'assertive');
-    toastElement.setAttribute('aria-atomic', 'true');
+  const toastElement = document.createElement("div");
```

Comment:
Do you have a linter we could all install? It would be nice to have one tbh...

Reply:
I use prettier for javascript. I can setup a prettier configuration for our repos that will avoid things like this.

Reply:
Please yeah, any one of them works, as long as it's easy to use.